texext/tests/plotdirective/conf.py

```diff
@@ -104,7 +104,7 @@
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-if LooseVersion(sphinx.__version__) >= LooseVersion('1.3'):
+if parse(sphinx.__version__) >= parse('1.3'):
```

Comment:
```suggestion
if pkgv.parse(sphinx.__version__) >= pkgv.parse('1.3'):
```

Just to be more explicit about what "parse" is, because it's such a generic name.